ionic_user_interface/src/components/header/Header.vue

```diff
@@ -30,21 +30,27 @@ export default defineComponent({
     IonToolbar,
     LoginButton,
   },
-  setup() {
-    const publicPath = process.env.BASE_URL;
-    return {
-      publicPath,
-    };
-  },
 });
 </script>
 
 <style scoped>
 .logo {
-  width: 40px;
-  height: 40px;
+  width: 60px;
+  height: 60px;
 }
 .logo:hover {
   cursor: pointer;
 }
+
+/* 
+TODO: Accomodate for various BASE_URL '/' vs '/RouteMaker/' etc
```

Comment:
Minor problem. I have googled a while and not sure how to solve this but does not impact the production environment. (Only the github page version). I will just create a new issue since not urgent?

Reply:
Simple fix would be to just add the CSS programatically lol

Reply:
I have tried exposing the path as a CSS variable as shown [here](https://www.telerik.com/blogs/passing-variables-to-css-on-a-vue-component). But to load the image in CSS I'll have to do something like 
```
.logo {
  content: url(var(--image-path))
}
```
But due to [legacy reasons](https://stackoverflow.com/questions/42330075/is-there-a-way-to-interpolate-css-variables-with-url) or something, it is not possible to interpolate CSS variables with `url()` currently.

Reply:
I mean using js lmaoo